server/src/main/java/org/opensearch/gateway/BaseGatewayShardAllocator.java

```diff
@@ -64,8 +69,9 @@ public abstract class BaseGatewayShardAllocator {
      * Allocate an unassigned shard to nodes (if any) where valid copies of the shard already exist.
      * It is up to the individual implementations of {@link #makeAllocationDecision(ShardRouting, RoutingAllocation, Logger)}
      * to make decisions on assigning shards to nodes.
-     * @param shardRouting the shard to allocate
-     * @param allocation the allocation state container object
+     *
```

Comment:
Is this applied by spotless fix ?

Reply:
Yes

Reply:
Let's not make unrelated changes. I don't think spotless would have made these changes as PRs are frequently being pushed and spotless is ensured in same. Since, this is old code, I highly doubt if there were spotless violations.

Reply:
Earlier I had added NodeShardState classes here, I guess these changes got flagged when I made that change in the file. I have moved those classes to PSA, I can remove this diff as this will most probably pass spotless check now.